dev/com.ibm.ws.crypto.ltpakeyutil/test/com/ibm/ws/crypto/ltpakeyutil/LTPACryptoTest.java

```
@@ -31,21 +31,38 @@ public class LTPACryptoTest {
                                      -113, -79, -22, -89, -21, 127, 39, -111, 21, -107, -91, 91, 26, -56, -123, -79, -10, 58, 39, -116, 82, 113, -6, -10, -20, -117, -106, -95, -34,
                                      -6, 37, -58, -22, -43, 89, -105, -65, 1, 0, 1 };
+    
+    private static final byte[] PUBLIC_KEY2 = new byte[] {0, -39, 7, -105, -87, 11, 13, 30, 127, -51, -53, 66, -31, -45, -37, 46, 49, -9, -99, 117, 52, -23, -99, 31, 69, -12, -59, 111, -33, -35, 124, 87, -49, -121, -40, 53, 39, 100, 67, 18, 33, -57, -122, 64, 51, -78, 36, 122, 2, 121, 93, -67, -128, -111, -65, 21, -30, -8, -100, -67, -13, -59, 72, 119, 40, -37, -2, -10, 37, 22, 104, 13, 108, 122, 124, -93, -6, 8, -70, -74, -7, 41, -117, 44, 52, 9, 11, -104, 98, -88, 67, 123, 5, -81, -76, 20, -35, -75, -114, -73, -45, -94, -28, -53, -51, 122, -125, -82, -61, -23, 15, 64, 8, 63, 42, 115, -31, -60, 101, -9, -70, 2, -27, -106, -61, -42, 22, -76, -11, 1, 0, 1};
 
-	private static final byte[] SECRET_KEY = new byte[] { 85, -21, 95, 39, 70, -51, -43, 15, -25, 8, 18, -94, -108, 32, -57, -25, -29, 37, 116, -8, -5, -83, -13, 83 };
-
+    private static final byte[] SECRET_KEY = new byte[] { 85, -21, 95, 39, 70, -51, -43, 15, -25, 8, 18, -94, -108, 32, -57, -25, -29, 37, 116, -8, -5, -83, -13, 83 };
     private static final String AES_CBC_CIPHER = "AES/CBC/PKCS5Padding";
     private static final String AES_ECB_CIPHER = "AES/ECB/PKCS5Padding";
     private static final String DES_CBC_CIPHER = "DESede/CBC/PKCS5Padding";
     private static final String DES_ECB_CIPHER = "DESede/ECB/PKCS5Padding";
 
     private static final byte[] ORIGINAL_DATA;
 
+    private static byte[] b;
+
+    //function to create new data for signing and verifying
+    private byte[] original_data(int i) throws Exception{
+        
+        try {
+            b = ("It's a original data."+Integer.toString(i)).getBytes("utf-8");
+        } catch (Exception e) {
+            b = null;
+        }
+    
+
+        return b;
+    }
```

i think it'd be more conventional to call this method `getOriginalData`.

also, i think we can just reduce this method to just:
```
    private byte[] getOriginalData(int i) throws Exception {
        return ("It's a original data." + Integer.toString(i)).getBytes("utf-8");
    }
```

since i think it might be easier for us to debug the error in the future if we throw the exception rather than return null and try to figure out where the null is coming from.

---

done